src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -506,23 +496,21 @@ CHIP_ERROR BLEManagerImpl::MapBLEError(int bleErr)
 
 void BLEManagerImpl::DriveBLEState(void)
 {
-    ChipLogProgress(DeviceLayer, "DriveBLEState starting");
+    SILABS_LOG("%s starting", __func__);
```

Comment:
I recommend stepping away from using SILABS_LOG except if it is something on the app side.

Also, this print doesn't provide much except specific debugging of the function flow. I suggest removing this trace for all together.

Reply:
Removed the log